### PR TITLE
Update BUILD.md to add ADB install

### PR DESCRIPTION
**Pull request by zadi15**

### CODE REVIEW

**Comment by zadi15** [comment]
Some people may not have adb installed by default and may not read the part that says that adb is required, so this just covers that area

File: BUILD.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ sudo apt install ffmpeg libsdl2-2.0-0
 # client build dependencies
 sudo apt install gcc git pkg-config meson ninja-build \
                  libavcodec-dev libavformat-dev libavutil-dev \
-                 libsdl2-dev
+                 libsdl2-dev adb
```
Comment (rom1v) [inline BUILD.md]: `adb` is a runtime dependency (not a "build dependency"), so it should be added few lines above instead :wink:

Comment (zadi15) [reply BUILD.md]: Ah, my apologies, I've just ammended it

**Comment by zadi15** [comment]
Moved `adb` into the runtime dependency area

**Comment by rom1v** [comment]
Rebased, squashed and merged on `master`: 684e0abb74cf2f5291d2c52bbf758e85f917a4d6

Thank you :+1:

**Pull request closed without merging**